draft-packages/collapsible/KaizenDraft/ExpertAdviceCollapsible/ExpertAdviceCollapsible.module.scss

```diff
@@ -0,0 +1,50 @@
+@import "~@kaizen/design-tokens/sass/border";
+@import "~@kaizen/design-tokens/sass/color";
+@import "~@kaizen/design-tokens/sass/spacing";
+
+$heading-active-color: $color-purple-100;
+
+.expertAdviceContainer {
+  background-color: $color-purple-100;
+  border: 1px $border-solid-border-style $color-purple-300;
+  color: $color-purple-800;
+  > div {
+    &:first-of-type {
```

Comment:
You'll need to run the linter to fix the indented styles here :)

Reply:
Run and now fixed.